bytecomp/translcore.ml

```diff
@@ -1021,7 +1035,7 @@ and transl_exp0 e =
       | `Constant_or_function ->
         (* a constant expr of type <> float gets compiled as itself *)
          transl_exp e
-      | `Float -> 
+      | `Float ->
           (* We don't need to wrap with Popaque: this forward
```

Comment:
Whilst looking at this PR I noticed the (unrelated) thing that maybe this comment is wrong when the float array optimization is disabled, as it may be in 4.06 onwards.  @chambart ?

Reply:
I don't know if this is the case, but this might change. We should indeed take the safe way here.